### PR TITLE
Fix rootHttpRouter service example

### PR DESCRIPTION
**Pull request by zerok**
Hey 🙂  The example in the rootHttpRouter service documentation indicates that the router also exposes a `use(handler)` method, which is not the case. This PR updates that example.

#### :heavy_check_mark: Checklist

- ~[ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))~
- [x] Added or updated documentation
- ~[ ] Tests for new functionality and regression tests for bug fixes~
- ~[ ] Screenshots attached (for UI changes)~
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: docs/backend-system/core-services/root-http-router.md [hunk 1]
```diff
@@ -27,11 +27,11 @@ createBackendPlugin({
       },
       async init({ rootHttpRouter }) {
         const router = Router();
-        router.get('/health', (request, response) => {
+        router.get('', (request, response) => {
```
Comment (benjdlambert) [inline docs/backend-system/core-services/root-http-router.md:30]: Maybe we should remove the `''` entirely, or use `*` or something a little better. It's a little confusing IMO in the docs to just have an empty string.

Alternatively, we could use something like `/readiness` or `/liveness` as they're common patterns in k8s to have different paths for different healthchecks.

**Review by benjdlambert — CHANGES_REQUESTED** [review]
Thanks for this, just a little nit that I think we wanna fix 👍

File: docs/backend-system/core-services/root-http-router.md (hunk 1 above)
Comment (zerok) [reply docs/backend-system/core-services/root-http-router.md]: Went with `/health/readiness` to make it more explicit :)

**Review by benjdlambert — APPROVED** [review]
Nice thanks :tada:

**Comment by zerok** [comment]
@benjdlambert Thanks 🙂

**Pull request merged**